neon-animation-runner-behavior.html

```diff
@@ -32,7 +32,7 @@
             try {
               result = neonAnimation.configure(config);
               // Check if we have an Effect rather than an Animation
-              if (typeof result.cancel != 'function') { 
+              if (!result || typeof result.cancel != 'function') { 
```

Comment:
sorry i accidentally removed my previous comment :x But it was basically questioning if `result = 0 | ''` are valid 
results for `document.timeline.play`. I guess this should be reverted no?